kong-ngx-build

```diff
@@ -329,11 +338,17 @@ main() {
           "--with-http_realip_module"
           "--with-http_stub_status_module"
           "--with-http_v2_module"
-          "--with-stream_ssl_preread_module"
-          "--with-stream_realip_module"
           "-j$NPROC"
         )
 
+        if version_gte $OPENRESTY_VER 1.11.4; then
+          OPENRESTY_OPTS+=('--with-stream_realip_module')
+        fi
+
+        if version_gte $OPENRESTY_VER 1.11.5; then
+          OPENRESTY_OPTS+=('--with-stream_ssl_preread_module')
+        fi
+
```

Comment:
It is equivalent, but should these checks be against `$NGINX_CORE_VER` instead?

Reply:
@eskerda They sure can! I will reorder the commits and slightly change the fix to use it :+1: